### PR TITLE
IGVF-1122-add-result-columns-to-multireport

### PR DESCRIPTION
**Pull request by mingjiecn**

### CODE REVIEW

File: src/igvfd/tests/indexing/test_searches_fields.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    rrf = ResultColumnsResponseField()
+    assert isinstance(rrf, ResultColumnsResponseField)
+
+
+def test_searches_fields_result_columns_response_field_query(workbook, testapp):
+    r = testapp.get(
+        '/multireport/?institute_label=Stanford'
+    )
+    assert 'result_columns' in r.json
+    assert len(r.json['result_columns']) < len(r.json['columns'])
```
Comment (keenangraham) [inline src/igvfd/tests/indexing/test_searches_fields.py:16]: Can we also have a test that expects some specific columns just to make sure we are getting what we want?

File: src/igvfd/searches/fields.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+
+def get_abstract_types(request):
+    types = []
+    item_registry = request.registry[TYPES]
+    for name, item in item_registry.abstract.items():
+        subtypes = item.subtypes
+        if len(subtypes) > 1:
+            types.append(name)
+    return types
```
Comment (keenangraham) [inline src/igvfd/searches/fields.py:88]: Can we keep this stuff in `report.py` for now and import it here? These deserve more clean up and refactoring if we move, and think `fields.py` should only have `Fields`.

**Review by keenangraham — CHANGES_REQUESTED** [review]
Looks good, just a couple things.

File: src/igvfd/searches/fields.py (hunk 2 above)
Comment (mingjiecn) [reply src/igvfd/searches/fields.py:88]: Put those functions in report.py or search_views.py will cause circular import so I have to put them here in fields.py

File: src/igvfd/tests/indexing/test_searches_fields.py (hunk 1 above)
Comment (mingjiecn) [reply src/igvfd/tests/indexing/test_searches_fields.py:16]: sure

File: src/igvfd/searches/fields.py (hunk 2 above)
Comment (keenangraham) [reply src/igvfd/searches/fields.py:88]: Still need better solution for this.

File: src/igvfd/tests/indexing/test_searches_fields.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    assert 'pi' in r.json['result_columns']
+    assert 'institute_label' in r.json['result_columns']
```
Comment (keenangraham) [inline src/igvfd/tests/indexing/test_searches_fields.py:18]: Why not test for all of the fields that are expected?

**Review by keenangraham — CHANGES_REQUESTED** [review]

File: src/igvfd/report.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 from snovault.elasticsearch.searches.interfaces import SEARCH_CONFIG
 from snosearch.parsers import QueryString
 from snovault.compat import bytes_
-from igvfd.search_views import search_generator
+from igvfd.search_generator import search_generator
```
Comment (keenangraham) [inline src/igvfd/report.py:8]: Looks good but can we put this in `/searches/generator.py`?

**Review by keenangraham — APPROVED** [review]

**Review by keenangraham — APPROVED** [review]
LGTM.

**Pull request merged**